modules/tinymce/src/core/main/ts/api/html/Schema.ts

```diff
@@ -571,7 +571,7 @@ const Schema = (settings: SchemaSettings = {}): Schema => {
 
   const isBlock = (name: string): boolean => Obj.has(getBlockElements(), name);
 
-  const isInline = (name: string): boolean => isValid(name) && !isBlock(name);
+  const isInline = (name: string): boolean => !Strings.startsWith(name, '#') && isValid(name) && !isBlock(name);
```

Comment:
Could you add tests for this in the schema tests so that the isBlock, isInline and isWrapper is feed both valid and invalid and #text etc. So that we make sure that it's the right thing going forward.